docs/manual/key-storage/storage-plugins/thycotic-storage.md

```diff
@@ -12,21 +12,28 @@ Process Automation users have access to the Thycotic Storage Plugin which can be
 
 Below is an example configuration, which can be configured using the *System Configuration* module. Add each setting as a configuration entry.
 
-![Thycotic Configuration](@assets/img/thycotic-config-screen.png)
+:::warning
+Doing it via Configuration Manager Plugin (GUI way to write configuration) will expose sensible fields such as the password. We recommend to use "$RDECK_BASE/rundeck-config.properties" file.
```

Comment:
This is not the recommendation.  We can enter a ticket to protect the Thycotic password, but setting via rundeck-config is still not the suggested method.

Reply:
ok! i'll remove it